ha/resource/resource_agent.py

```diff
@@ -192,14 +192,8 @@ def stop(self):
         Log.debug(f"In stop for {filename}")
         if os.path.exists(const.HA_INIT_DIR + filename):
             os.remove(const.HA_INIT_DIR + filename)
-        while True:
-            if self.monitor() == const.OCF_NOT_RUNNING:
-                time.sleep(2)
-                break
-        if self.monitor(state=const.STATE_STOP) == Action.RESTART:
-            Log.debug(f"Restarting {filename} resource")
-            time.sleep(2)
-            return const.OCF_SUCCESS
+            Log.debug(f"Stopping {filename} resource")
```

Comment:
It seems like this log is misleading. It will log if and only if there is a file to remove (so if it logs this message, then the file has just been removed - but that is not the user will read in the logs). And moreover, immediately after this log line there is another log - Stopped {filename} resource. Between those log messages there is nothing time-consuming or at least anything that can fail.